config/config.go

```diff
@@ -83,7 +83,6 @@ type NodeConfig interface {
 
```

Comment:
i think for experiments and gamma and hackathons its a good idea to avoid the sanitisation ... i would not like to see this feature go away yet.

Reply:
I agree with Noam, sometimes it is necessary to remove the restrictions, especially during the hackathons.

I recommend keeping the option.

Reply:
ok